sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1629,7 +1629,7 @@ object SQLConf {
       .createWithDefault(true)
 
   val PANDAS_ARROW_SAFE_TYPE_CONVERSION =
-    buildConf("spark.sql.execution.pandas.arrowSafeTypeConversion")
+    buildConf("spark.sql.execution.pandas.arrowSafeTypeConversion.enabled")
```

Comment:
`arrowSafeTypeConversion` does not have sub-configs, is it possible to make it a verb?

Reply:
updated with `convertToArrowArraySafely`.